index.html

```diff
@@ -787,8 +787,16 @@ <h2>Links</h2>
 
       <p>
         <span class="rfc2119-assertion" id="common-constraints-links-1">The following keywords are defined for the
-          HTTP profiles and MUST be supported by all consumers.</span>
+          HTTP profiles and MAY be present in compliant TDs.</span>
```

Comment:
I'm not sure what "compliant" means here. Does also "non-compliant" TD exist?  Or do you mean "profile-enabled TDs"?

Reply:
Should be "profile compliant" - changed.